src/components/gallery/HdGalleryMedia.vue

```diff
@@ -25,6 +25,7 @@
           'background-image': `url('${item.thumbnail}')`,
         }"
       />
+      <a v-if="link" class="gallery-media__object__link" :href="link"></a>
```

Comment:
What if navigation is handled with vue router?
I think would be safer to implement an @click=$emit() and then we can handle redirection from a method.
Also, we need a unit test for this scenario.

Reply:
working on it